guides/common/modules/ref_puppet-additional-resources.adoc

```diff
@@ -1,5 +1,5 @@
 [id="puppet_guide_additional_resources_{context}"]
 = Additional Resources for Puppet
 
-* The https://puppet.com/docs/open-source-puppet[official Puppet documentation] is a good starting point.
+* The https://puppet.com/docs/puppet/7/puppet_index.html[official Puppet documentation] is a good starting point.
```

Comment:
You can drop the change and rebase to master; see https://github.com/theforeman/foreman-documentation/commit/d35a944a859dea69ff9314639dfa2fdd6f825ef1